src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,266 @@
+package org.broadinstitute.hellbender.tools.genomicsdb;
+
+import com.intel.genomicsdb.ChromosomeInterval;
+import com.intel.genomicsdb.GenomicsDBImporter;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.FeatureCodec;
+import htsjdk.tribble.FeatureReader;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFCodec;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.GenomicsDBProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureInput;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.*;
+
+/**
+ * This tool imports GVCFs to GenomicsDB.
+ * User must specify a loader JSON configuration file,
+ * callsets JSON file containing the stream names for samples,
+ * and a stream JSON file containing the filenames of the streams
+ *
+ * Use the GenomicsDB query interface for reading variants from
+ * the database
+ */
+@CommandLineProgramProperties(
+  summary = "Import VCFs to GenomicsDB",
+  oneLineSummary = "Import VCFs to GenomicsDB",
+  programGroup = GenomicsDBProgramGroup.class
+)
+public final class GenomicsDBImport extends GATKTool {
+
+  private static final long serialVersionUID = 1L;
+  private final long DEFAULT_SIZE_PER_COLUMN_PARTITION_PER_SAMPLE = 16*1024L;
+  private final long DEFAULT_SEGMENT_SIZE = 1048576L;
+  private final String DEFAULT_ARRAY_NAME_PREFIX = "genomicsdb_array";
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_WORKSPACE_LONG_NAME,
+    shortName = StandardArgumentDefinitions.GENOMICSDB_WORKSPACE_SHORT_NAME,
+    doc = "Workspace where the database will be persisted. " +
+      "Must be either a local path or full path of a " +
+      "parallel file system like Lustre or NFS")
+  private String workspace = "";
```

Comment:
Don't initialize this, so that the argument system will be able to detect a missing value for you.